linux/Platform.c

```diff
@@ -146,6 +146,7 @@ static enum { BAT_PROC, BAT_SYS, BAT_ERR } Platform_Battery_method = BAT_PROC;
 static time_t Platform_Battery_cacheTime;
 static double Platform_Battery_cachePercent = NAN;
 static ACPresence Platform_Battery_cacheIsOnAC;
+static unsigned long long int prevResidueTime, curResidueTime;
```

Comment:
When moving these two variables to platform code, I suggest renaming these to `prevGPUResidueTime` and `curGPUResidueTime` to reduce ambiguity.

By the way, `curGPUResidueTime` may be converted into a local (function scope) variable, but that's a separate commit.